src/microbe_stage/ProcessList.cs

```diff
@@ -12,6 +12,19 @@ public class ProcessList : VBoxContainer
 
     public List<IProcessDisplayInfo> ProcessesToShow { get; set; }
 
+    public bool ShowSpinners { get; set; } = true;
+
+    /// <summary>
+    ///   The default color for all the processes title in this list.
+    /// </summary>
+    public Color ProcessesTitleColour { get; set; } = Colors.White;
+
+    /// <summary>
+    ///   Controls whether the color of all processes title in this list should be turned
```

Comment:
I think this comment is unclear, does this mean:
- The individual process name turns red if it has limiting compounds
- Or all of the processes turn red if any has limiting compounds
?